src/content/docs/logs/forward-logs/circleci-logs.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Forward CircleCI logs to New Relic"
+metaDescription: "Integrate your CircleCI data with New Relic observability."
+tags:
+  - CircleCI
+  - Instant Observability
+  - Logs
+  - Logpush
+  - CICD
+---
+
+Configure the [CircleCI webhook](https://circleci.com/docs/2.0/webhooks/) service to send CI/CD Logs to New Relic.
+
+## Set up a webhook trigger in CircleCI [#setup]
+
+Complete these steps to set up a CircleCI webhook to forward your logs to New Relic:
+
+1. Log in to [CircleCI](https://app.circleci.com/projects).
+
+2. Go to one of your CircleCI projects.
+
+3. Click **Project Settings**.
+
+4. In the sidebar of your Project Settings, click **Webhooks**.
+
+5. Click **Add Webhook**.
```

Comment:
```suggestion
5. Click **Add webhook**.
```

Reply:
These are CircleCI UI things. I have to trust that they're capitalized correctly.

Reply:
Oh this is an interesting situation! Our style guide says we `use sentence case and bold for UI elements, even if the UI element is in a different case in the UI`, but maybe that should only apply to _our_ product and not when we reference other products?